### DISCUSSION TITLE
Daily Standup - Wednesday, April 19

### DISCUSSION THREAD

**Discussion by antonieto** [antonieto/ibm-dashboard#134] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by paotg123** [comment]
**What did you achieve in the last 24 hours?**:
 - Ver tutorial de login/sign in 

**What are your priorities for the next 24 hours?**:
 - comenzar con front de lo mencionado 

**Blockers**:
 -¿autenticación?

**Shoutouts**:
 - @username for @everyone

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Not much

**What are your priorities for the next 24 hours?**:
 - Define DB tasks with @Int-Angel 

**Blockers**:
 - Have been a little busy with work

**Shoutouts**:
 - @everyone for being such a good team

**Comment by Int-Angel** [comment]
**What did you achieve in the last 24 hours?**:
 - took a breack

**What are your priorities for the next 24 hours?**:
 - define tasks
 - deploy db

**Blockers**:
 - none

**Shoutouts**:
 - @username for @DavBot02

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Started mini course for passportjs auth with JWT

**What are your priorities for the next 24 hours?**:
 - Finish course

**Blockers**:
 - None

**Shoutouts**:
 - @Jhoscar22

**Reply by antonieto** [reply]
JWT Auth video: https://www.youtube.com/watch?v=F-sFp_AvHc8

**Comment by Jhoscar22** [comment]
**What did you achieve in the last 24 hours?**:
 - Introduction to passport

**What are your priorities for the next 24 hours?**:
 - Sign-up with email

**Blockers**:
 - None

**Shoutouts**:
 - @everyone
